src/aap_eda/settings/default.py

```diff
@@ -636,6 +643,17 @@ def get_rulebook_process_log_level() -> RulebookProcessLogLevel:
 # Organization and object roles will come from create_initial_data
 ANSIBLE_BASE_ROLE_PRECREATE = {}
 
+# --------------------------------------------------------
+# GATEWAY RESOURCE API CLIENT
+# --------------------------------------------------------
+RESOURCE_SERVER = {
+    "URL": settings.get("RESOURCE_SERVER__URL", "https://localhost"),
+    "SECRET_KEY": settings.get("RESOURCE_SERVER__SECRET_KEY", ""),
```

Comment:
Should this be our already defined SECRET KEY?

Reply:
no, it must be generated by the gateway